src/dispatch.rs

```diff
@@ -1,20 +1,34 @@
 use std::ffi::OsString;
 use std::path::Path;
 
+use serde::{Deserialize, Serialize};
+use serde_urlencoded;
```

Comment:
Nit: Is this necessary if you're going to use a qualified function like `serde_urlencoded::to_string`?

```suggestion
```